diff --git a/Trunk.toml b/Trunk.toml
index f3704bf..b6e2f69 100644
--- a/Trunk.toml
+++ b/Trunk.toml
@@ -6,7 +6,7 @@ release = true
 # The output dir for all final assets.
 dist = "dist"
 # The public URL from which assets are to be served.
-public_url = "https://oxide-chess.fly.dev/"
+public_url = "http://localhost:8080/"
 # Whether to include hash values in the output file names.
 filehash = true
 
diff --git a/client/src/game_socket.rs b/client/src/game_socket.rs
index 1aa6f42..167027c 100644
--- a/client/src/game_socket.rs
+++ b/client/src/game_socket.rs
@@ -30,7 +30,7 @@ pub(super) async fn create_game_socket(
 }
 
 async fn connect_to_socket(game_id: u32) -> anyhow::Result<(WriteStream, ReadStream)> {
-    let url = format!("wss://oxide-chess.fly.dev/game/{game_id}");
+    let url = format!("ws://localhost:8080/game/{game_id}");
     Ok(connect(Url::parse(&url)?).await?.split())
 }
 
diff --git a/client/src/launcher.rs b/client/src/launcher.rs
index d69f7a2..61cb989 100644
--- a/client/src/launcher.rs
+++ b/client/src/launcher.rs
@@ -5,7 +5,7 @@ pub fn launch() {
         use dioxus_fullstack::prelude::server_fn;
 
         log::info!("configuring desktop..");
-        server_fn::set_server_url("https://oxide-chess.fly.dev");
+        server_fn::set_server_url("http://localhost:8080");
 
         dioxus_desktop::launch_cfg(
             crate::components::App,
