diff --git a/Trunk.toml b/Trunk.toml
index d2a4233..b6e2f69 100644
--- a/Trunk.toml
+++ b/Trunk.toml
@@ -6,7 +6,7 @@ release = true
 # The output dir for all final assets.
 dist = "dist"
 # The public URL from which assets are to be served.
-public_url = "https://muddy-fog-684.fly.dev/"
+public_url = "http://localhost:8080/"
 # Whether to include hash values in the output file names.
 filehash = true
 
diff --git a/client/src/game_socket.rs b/client/src/game_socket.rs
index bdac8e9..c7fe372 100644
--- a/client/src/game_socket.rs
+++ b/client/src/game_socket.rs
@@ -23,7 +23,7 @@ pub async fn create_game_socket(game: UseRef<Game>, game_id: u32, rx: UnboundedR
 }
 
 async fn connect_to_socket(game_id: u32) -> anyhow::Result<(WriteStream, ReadStream)> {
-    let url = format!("wss://muddy-fog-684.fly.dev/game/{game_id}");
+    let url = format!("ws://localhost:8080/game/{game_id}");
     Ok(connect(Url::parse(&url)?).await?.split())
 }
 
diff --git a/client/src/main.rs b/client/src/main.rs
index 0a411a0..9f244e6 100644
--- a/client/src/main.rs
+++ b/client/src/main.rs
@@ -13,7 +13,7 @@ pub fn main() {
         use dioxus_desktop::{Config, LogicalSize, WindowBuilder};
         use dioxus_fullstack::prelude::server_fn::set_server_url;
         const WINDOW_SIZE: u32 = 800;
-        set_server_url("https://muddy-fog-684.fly.dev");
+        set_server_url("http://localhost:8080");
         log::info!("desktop launching");
         dioxus_desktop::launch_cfg(
             App,
